newdle/schemas.py

```diff
@@ -172,6 +172,11 @@ def validate_timeslots(self, v):
 
 class UpdateNewdleSchema(NewNewdleSchema):
     participants = fields.List(fields.Nested(NewParticipantSchema, unknown=EXCLUDE))
+
+
+class NewEventSchema(mm.Schema):
+    title = fields.String(validate=validate.Length(min=3, max=80), required=True)
+    duration = fields.Integer()
     final_dt = fields.DateTime(format=DATETIME_FORMAT)
```

Comment:
Look closely at the change here:  you are basically removing the `final_dt` from `UpdateNewdleSchema` which is what's breaking the tests. Add it back and the tests should pass. And maybe rename it to `start_dt` in here since this is more descriptive of what it is used for in NewEventSchema.

Reply:
Ah, I missed that! (I've now removed the `NewEventSchema`.)